canonical-web-and-design/vanilla-framework
# Add password toggle pattern

## Done
Added a pattern for showing and hiding the value of password fields

## QA
- Open [demo](https://vanilla-framework-3899.demos.haus/docs/examples/patterns/forms/password-toggle)
- Check that the pattern matches [the spec](https://docs.google.com/document/d/1Q5i2DI801rKA4XO8MZv0dldUUGGbB6zPSWsRVLf8mk0/edit)
